src/app/room-creation/services/random-names-provider.service.spec.ts

```diff
@@ -25,4 +25,13 @@ describe('RandomNamesProvider', () => {
 
     service.provide('Imene').subscribe(name => expect(name).toBe('Daniel'));
   });
+
+  it('initializes the names and does not make another call when providing them afterwards', () => {
+    spyOn(httpClient, 'get').and.returnValue(of(['Pollin']));
```

Comment:
what about having an empty line to separate Arrange-Act-Assert? it was your thing!!!

Reply:
Well done!